src/components/FilterPanel.jsx

```diff
@@ -70,12 +100,36 @@ export default function FilterPanel({ onClose }) {
             expandIcon={<ExpandMoreIcon />}
             aria-controls={name + '-content'}
             id={name + '-header'}>
-            <Typography>{name}</Typography>
+            <Grid container alignItems="center" justifyContent="space-between">
+              <Typography>{name}</Typography>
+              <Badge
+                badgeContent={filter.badge}
+                color="primary"
+                sx={{ mr: 2 }}
+              />
+            </Grid>
           </AccordionSummary>
 
-          <AccordionDetails sx={{ m: 1 }}>{filter}</AccordionDetails>
+          <AccordionDetails sx={{ m: 1 }}>{filter.comp}</AccordionDetails>
         </Accordion>
```

Comment:
What do you think of surrounding `<Accordion>`'s children with a `<Container maxWidth='sm`>`? That way on screens just barely smaller than `md` won't have such a large distance between the filter name and the number of filters applied. It would also help shorten the width of the Majors autocomplete a bit which you were concerned about last time:
<img width="898" alt="Screen Shot 2022-02-05 at 4 07 54 PM" src="https://user-images.githubusercontent.com/8023233/152663178-83e126b6-4eeb-4516-906c-d8e3d792f3eb.png">

Reply:
Doesn't it look a bit off? What if we use a custom width that is shorter than `xs`
<img width="808" alt="Screen Shot 2022-02-05 at 9 04 27 PM" src="https://user-images.githubusercontent.com/40483569/152668414-1126db9b-62f8-45a5-9e5a-f5aff1f6b0cd.png">

Reply:
You need to surround both the summary and the detail component in the Container (or maybe have containers within them)

Reply:
Ok that looks a bit better. I had to surround the Summary and the Details in different containers. 

<img width="868" alt="Screen Shot 2022-02-06 at 11 22 21 AM" src="https://user-images.githubusercontent.com/40483569/152697809-6f71b8ff-f317-483e-a8a7-71b837ad64be.png">

Reply:
Strange it had to be separate containers 🤔

Reply:
Add `disableGutters` to the containers so it doesn't have left and right padding

Reply:
yeah if I put them in the same container, the Accodrion would stay open and the drop down wouldn't work.